docs/readmes/basics/prerequisites.md

```diff
@@ -58,7 +58,7 @@ Development can occur from multiple OS's, where **macOS** and **Ubuntu** are **e
 ### Ubuntu
 
 1. Install the following tools
-   1. [Docker](https://docs.docker.com/engine/install/ubuntu/) and [Docker Compose](https://docs.docker.com/compose/install/)
+   1. [Docker and Docker Compose](https://docs.docker.com/engine/install/ubuntu/)
```

Comment:
Why is the desktop version linked for Mac but the engine for Ubuntu? https://docs.docker.com/desktop/install/ubuntu/ would be the equivalent for line 28 I think.

Reply:
The distribution and payment options are different for Linux and Windows.
On Windows there basically is only "Docker Desktop" to install.
On Linux you do not need the GUI of "Docker Desktop", but only having the basic engine is enough for docker to work. So with that regard it is the minimal requirement, and the GUI is an optional addition, but nor required.

Reply:
That is correct. There is no Docker Engine for macOS, but installing the Docker GUI if not needed seemed an overkill to me.